local/build_api_executors.py

```diff
@@ -72,6 +56,13 @@ def make_default_script(script):
     req_type = request_data["Method"]
     req_api = request_data["Query"]
     req_params = request_data["Params"]
+
+    content_type=""
```

Comment:
it'd better leverage special function here and in `build_api_services` and in rest-api scripts